docs/advanced/kiali.adoc

```diff
@@ -0,0 +1,187 @@
+jaeger-query           jaeger-query-istio-system.127.0.0.1.nip.io                     jaeger-query           jaeger-query      edge          None
+kiali                  kiali-istio-system.127.0.0.1.nip.io                            kiali                  <all>                           None
+prometheus             prometheus-istio-system.127.0.0.1.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.127.0.0.1.nip.io                          tracing                tracing           edge          None
+----
+
+So now we have the URL for Kiali (usually,
+"kiali-istio-system.127.0.0.1.nip.io"), so let's access it:
+
+image:docs/images/kiali-login.png[Login Page]
+
+The default credentials are "admin/admin", but it's recommended to change them
+before using it in production.
+
+== Generating Sample Data
+
+To show the capabilities of Kiali, you'll need an Istio-enabled application to
+be running. For this, we can use the `customer-tutorial` application we created
+earlier.
+
+To generate data for it, we can `curl` it with this command:
+
+[source, bash]
+----
+curl customer-tutorial.$(minishift ip).nip.io
+----
+
+== Service Graph
+
+After you login, you should see the Service Graph page:
+
+image:docs/images/kiali-service-graph.png[Service Graph Page]
+
+It shows a graph with all the microservices, connected by the requests going
+through then. On this page, you can see how the services interact with each
+other.
+
+== Service Listing
+
+Click the Services link in the left navigation. On the Services page you can
+view a listing of all the services that are running in the cluster, and
+additional information about them, such as health status.
+
+image:docs/images/kiali-service-list.png[Service Listing Page]
+
+Click on the "Preferences" service to view the Service Details page, where you
+can view additional information about each service.
+
+image:docs/images/kiali-service-details.png[Service Details]
+
+You can see the health status of the service (a Healthy service means one that
+is is up and responding correctly):
+
+image:docs/images/kiali-service-health.png[Health Status]
+
+You can also see the deployments:
+
+image:docs/images/kiali-deployments.png[Deployments]
+
+The source services:
+
+image:docs/images/kiali-source-services.png[Source Services]
+
+And the virtual services on the Istio infrastructure:
+
+image:docs/images/kiali-virtual-services.png[Virtual Services]
+
+== Istio Config
+
+Click the Istio Config link on the left navigation. On this page, you can see
+all currently running config rules, such as Virtual Services, Route Rules,
+Routes, Circuit Breakers, Fault Injection and so on.
+
+image:docs/images/kiali-istio-config.png[Istio Config Page]
+
+== Distributed Tracing
+
+Click on the Distributed Tracing link in the left navigation. On this page you
```

Comment:
Whe I click on "Distributed tracing" it shows the following error:  "Could not fetch Jaeger info Error: [ Unable to find Jaeger URL: no route defined. ExternalIPs not defined on service 'jaeger' ]" as you can see in the attached pic
<img width="1213" alt="screenshot 2018-08-07 19 10 10" src="https://user-images.githubusercontent.com/30131/43807254-9a627f34-9a75-11e8-941d-1beee8053651.png">

Reply:
This is fixed on latest master.